src/commands/config-set.json

```diff
@@ -19,6 +19,10 @@
             "LOADING",
             "STALE"
         ],
+        "command_tips": [
+            "REQUEST_POLICY:ALL_SHARDS",
```

Comment:
I would expect this to be something like "ALL_NODES" instead of "ALL_SHARDS", since you also want to set it on replicas.